cert/aziot-certd-config/src/lib.rs

```diff
@@ -30,6 +30,16 @@ pub struct Config {
     #[serde(default, skip_serializing)]
     #[cfg_attr(not(debug_assertions), serde(skip_deserializing))]
     pub endpoints: Endpoints,
+
```

Comment:
For consistency, move this above the `endpoints` field. (`endpoints` is always the last field.)

Reply:
Fixed